notebooks/docs.clj

```diff
@@ -0,0 +1,347 @@
+;; # 📓 Clerk Documentation
+(ns docs
+  {:nextjournal.clerk/toc true}
+  (:require [clojure.string :as str]
+            [next.jdbc :as jdbc]
+            [nextjournal.clerk :as clerk]
+            [nextjournal.clerk.parser :as parser]
+            [nextjournal.clerk.eval :as eval]
+            [nextjournal.clerk.analyzer :as ana]
+            [nextjournal.clerk.viewer :as v]
+            [weavejester.dependency :as dep]))
+
+;; ## ⚖️ Rationale
+
+;; ## 🚀 Getting Started
+
+;; ### ⏱ File Watcher
+
+;; ### 🔪 Editor Integration
+
+;; For even better flow states, we recommend you bind clerk/show! to a shortcut in your favorite editor:
```

Comment:
just suggestions around copy...

```suggestion
;; Clerk is designed to be used alongside your editor. To make this possible, bind `clerk/show!` to an editor keyboard shortcut.
```

I think it's just complicated enough to include links to per-editor documentation, e.g.

```suggestion
Here's some help setting that up with:
- [intellij & cursive](https://cursive-ide.com/userguide/repl.html#repl-commands)
```